app/invocation/invocation.tsx

```diff
@@ -251,6 +252,10 @@ export default class InvocationComponent extends React.Component<Props, State> {
             <ExecutionCardComponent model={this.state.model} inProgress={this.state.inProgress} />
           )}
 
+          {activeTab == "action" && (
+            <ActionCardComponent model={this.state.model} search={new URLSearchParams(window.location.search)} />
```

Comment:
This is already a prop of the invocation component - so you could just pass `search={this.props.search}` here.